include/DataProvider.hpp

```diff
@@ -0,0 +1,46 @@
+/*
+*   DataProvider.hpp
+*   ----------------------
+*   Created on: 2023/09/29
+*   Author: Lankow
+*/
+#ifndef DATA_PROVIDER_HPP
+#define DATA_PROVIDER_HPP
+
+#include <Arduino.h>
+#include "constants.hpp"
+
+class DataProvider
+{
+    private:
+    uint16_t m_currentHumidityLvl[MAX_SENSORS_NO];
```

Comment:
m_currentHumidityLvl
m_humidityThreshold
m_humidityActive

to wszystko parametry dla jednego sensora. Duuuuużo czyściej by było zrobić klase Sensor czy coś takiego z 3 parmasami i tutaj trzymać tablice tych klas